WPMedia/fusion-news-theme-blocks
# [PEN-1533] - article body shoring empty box if nullable

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
Fixed issue, when content of video are not defined it was shown Component Code Error:
![image](https://user-images.githubusercontent.com/47773457/101053667-9f6f6900-35ba-11eb-8618-9ca2dd39908d.png)

## Jira Ticket
- [PEN-1533](https://arcpublishing.atlassian.net/browse/PEN-1533)

## Acceptance Criteria
_copy from ticket_

## Test Steps

1. In Fusion-News-Theme repo, checkout master & git pull
2. After checking out this feature branch in fusion-news-theme-blocks, run rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/video-player-block && npm i && cd ../..
3. Run npx fusion start -f @wpmedia/video-player-block in Fusion-News-Theme
4. Create new page and add Feature 'Video Center Player'
5. Empty area will appear - meaning source video not provided
6. Set custom field Configure Content -> Schemas = ans-item -> Content Source=content-api -> website_url = /video/2019/12/12/castle-church-tower/
7. Refresh page && Video will appear as expected

## Effect Of Changes
### Before
If website_url not provided error box appearing

![image](https://user-images.githubusercontent.com/47773457/101055708-c2028180-35bc-11eb-87da-0024367ad7e9.png)

### After
If website_url not provided it will show an empty area

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
